permissions/src/androidTest/java/com/google/accompanist/permissions/MultiplePermissionsStateTest.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.accompanist.permissions
+
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.test.filters.SdkSuppress
+import androidx.test.rule.GrantPermissionRule
+import com.google.accompanist.permissions.test.PermissionsTestActivity
+import com.google.common.truth.Truth.assertThat
+import org.junit.Rule
+import org.junit.Test
+
+/**
+ * Test that proves the data comes from the right place
+ */
+@SdkSuppress(minSdkVersion = 23)
+class MultiplePermissionsStateTest {
+
+    @get:Rule
+    val composeTestRule = createAndroidComposeRule<PermissionsTestActivity>()
+
+    @get:Rule
+    val permissionRule = GrantPermissionRule.grant(
+        "android.permission.CAMERA",
+        "android.permission.READ_EXTERNAL_STORAGE"
+    )!!
+
+    @Test
+    fun permissionState_hasPermission() {
+        composeTestRule.setContent {
+            val state =
+                composeTestRule.activity.activityResultRegistry.rememberMultiplePermissionsState(
```

Comment:
Extract `composeTestRule.activity.activityResultRegistry` to a object val or fun? It's a bit long to read in every test.